src/calculator/votes.ts

```diff
@@ -30,18 +35,18 @@ export async function getVotesWithCoefficients(
   );
 
   const enablePassport =
-    options.enablePassport ??
-    round?.metadata?.quadraticFundingConfig?.sybilDefense ??
+    args.options.enablePassport ??
```

Comment:
Just FYI you can still enjoy named parameters here by deconstruction if you wish, as in:

```ts
export  async function getVotesWithCoefficients(
  { applications, options, rounds }: GetVotesWithCoefficientsArgs
): ... {
  // ...
  const enablePassport = options.enablePassport ?? ...
```

(But `args` is just as fine.)

Reply:
@bard yeah that was the initial implementation but it was very noisy, so I moved to args